Project.toml

```diff
@@ -10,7 +10,7 @@ Unitful = "1986cc42-f94f-5a68-af5c-568840ba703d"
 [compat]
 RecipesBase = "1.0"
 Unitful = "0.16, 0.17, 0.18, 1.0"
-julia = "1.0"
+julia = "1.0 - 1.8.1"
```

Comment:
Mhh I think we should not limit the version here? Otherwise when somebody does `]add UnitfulRecipes` on julia 1.9 say an old release without deprecation warning will get installed. Or do I see this wrong?

Reply:
I suggested this after reading https://discourse.julialang.org/t/how-do-i-deprecate-a-package/14129, but nevermind, archiving + throwing a warning should be sufficient.